proto/stream_service.proto

```diff
@@ -100,9 +109,14 @@ message StreamingControlStreamRequest {
     uint64 prev_epoch = 2;
   }
 
+  message RemovePartialGraphRequest {
```

Comment:
What is `RemovePartialGraphRequest` for?

Reply:
It's for cleaning the stale partial graph. I used to try to let CN clean the empty partial graph by itself, but to handle some corner cases, we may need more complicated actor state management in some previous [commit](https://github.com/risingwavelabs/risingwave/pull/17758/commits/6dbb67fe426dd189d813dbaa9b74458047eff7df). To make this PR simple, I finally chose to let meta explicitly send this request to clean the stale partial graph.

Reply:
> clean the empty partial graph by itself

What do you mean by "empty" partial graph?

Reply:
In the partial checkpoint based backfill implementation, each creating streaming job will have a separate partial graph. When streaming jobs finish creating, it will be merged to the original global streaming graph, and then the original partial graph will become empty and needs to be clear.